api/src/db/migrations/20250813204000_add-attribute-to-archive-items-table.ts

```diff
@@ -0,0 +1,17 @@
+import type { Knex } from "knex"
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.alterTable("archive_items", (table) => {
+    table.text("sharing_purpose").notNullable().defaultTo("n/a")
+    table.boolean("confidentiality_receipt").notNullable().defaultTo(false)
+    table.string("yukon_first_nations", 255).notNullable().defaultTo("n/a")
```

Comment:
`n/a` seems like something that should be in the UI rather than the database.
I would expect the default to be `""` and parse out to and empty array. See later comment on the model.